src/res/flame.html

```diff
@@ -221,14 +493,44 @@ <h1>/*title:*/</h1>
 				canvas.title = f.title + '\n(' + samples(f.width) + f.details + ', ' + pct(f.width, levels[0][0].width) + '%)';
 				canvas.style.cursor = 'pointer';
 				canvas.onclick = function() {
-					if (event.altKey && h >= root.level) {
+					if (event.altKey && h > root.level) {
```

Comment:
OK, "all" frame should never be removed, but otherwise what's wrong with deleting stack at current level?